crates/consensus/common/src/calc.rs

```diff
@@ -24,10 +24,10 @@ use reth_chainspec::{EthereumHardfork, Hardforks};
 pub fn base_block_reward(
     chain_spec: impl Hardforks,
     block_number: BlockNumber,
-    block_difficulty: U256,
-    total_difficulty: U256,
+    _block_difficulty: U256,
+    _total_difficulty: U256,
 ) -> Option<u128> {
-    if chain_spec.fork(EthereumHardfork::Paris).active_at_ttd(total_difficulty, block_difficulty) {
+    if chain_spec.fork(EthereumHardfork::Paris).active_at_ttd(block_number) {
```

Comment:
should I keep the active_at_ttd function as it is and add a new active_at_merge method instead?

Reply:
let's keep this for now to keept changes small